packages/date-picker/src/vaadin-date-picker-mixin.js

```diff
@@ -415,17 +414,11 @@ export const DatePickerMixin = (subclass) =>
     ready() {
       super.ready();
 
-      addListener(this, 'tap', (e) => {
+      this.addEventListener('click', (e) => {
         if (!this._isClearButton(e) && (!this.autoOpenDisabled || this._noInput)) {
           this.open();
         }
       });
-
-      this.addEventListener('touchend', (e) => {
-        if (!this._isClearButton(e)) {
-          e.preventDefault();
-        }
-      });
```

Comment:
Canceling the "touchend" events seems not to be relevant anymore. It was [originally added](https://github.com/vaadin/vaadin-date-picker/pull/39) 6 years ago and was probably related to preventing ghost clicks.